src/views/Entries/QualityControl/EntryCard/index.tsx

```diff
@@ -287,90 +297,116 @@ function EntryCard(props: EntryCardProps) {
+                            <div className={styles.confidentiality}>
+                                { lead.confidentialityDisplay }
+                            </div>
+                        </div>
+                        <div className={styles.entryDetailsRow}>
                             <div
-                                className={styles.value}
-                                title={_ts('entries.qualityControl', 'leadSourceTooltip', { leadSource })}
+                                className={styles.createdBy}
+                                title={_ts('entries.qualityControl', 'leadCreatedByTooltip', { user: entry.createdByName })}
                             >
-                                { lead.sourceDetail ? lead.sourceDetail.title : lead.sourceRaw }
+                                { entry.createdByName }
                             </div>
+                            <DateOutput
+                                className={styles.createdAt}
+                                value={entry.createdAt}
+                                tooltip={_ts('entries.qualityControl', 'entryCreatedOnTooltip')}
+                            />
                         </div>
-                        <div className={styles.confidentiality}>
-                            { lead.confidentialityDisplay }
-                        </div>
-                    </div>
-                    <div className={styles.entryDetailsRow}>
-                        <div
-                            className={styles.createdBy}
-                            title={_ts('entries.qualityControl', 'leadCreatedByTooltip', { user: entry.createdByName })}
-                        >
-                            { entry.createdByName }
-                        </div>
-                        <DateOutput
-                            className={styles.createdAt}
-                            value={entry.createdAt}
-                            tooltip={_ts('entries.qualityControl', 'entryCreatedOnTooltip')}
+                        <ListView
+                            className={styles.scaleWidgets}
+                            data={scaleWidgets}
+                            renderer={ListItem}
+                            rendererParams={scaleWidgetRendererParams}
+                            keySelector={widgetKeySelector}
+                            emptyComponent={EmptyComponent}
+                        />
+                    </section>
+                }
+                <div className={styles.actionsWrapper}>
+                    {!isVisible
+                        ? (
+                            <Button
+                                className={styles.expandButton}
+                                onClick={handleShow}
+                                transparent
+                            >
+                                <Icon name="chevronUp" />
+                                {_ts('entries', 'more')}
+                            </Button>
```

Comment:
Icons can be directly sent to Button
```suggestion
                            <Button
                                className={styles.expandButton}
                                onClick={handleShow}
                                iconName="chevronUp"
                                transparent
                            >
                                {_ts('entries', 'more')}
                            </Button>
```